jni/dc_wrapper.c

```diff
@@ -1302,7 +1302,7 @@ JNIEXPORT jstring Java_com_b44t_messenger_DcMsg_getOverrideSenderName(JNIEnv *en
     char* temp = dc_msg_get_override_sender_name(get_dc_msg(env, obj));
         jstring ret = NULL;
         if (temp) {
-            JSTRING_NEW(temp);
+            ret = JSTRING_NEW(temp);
```

Comment:
oh, what a nice bug. i added then when fixing [another one](https://github.com/deltachat/deltachat-android/pull/1781/commits/bf88f0de45a6071afeb2c108c06653636880a3d9) :) but now it should be fine :)